src/Symfony/Component/Clock/Clock.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+/**
+ * A global clock.
+ *
+ * Defined as enum to be final and non-instantiable.
+ */
+enum Clock
```

Comment:
As you target 6.3 which requires PHP >= 8.1, why not ?
```suggestion
final Clock
```

Reply:
this would make the class instantiable

Reply:
yep final class with private constructor does the job and it seems less weird but up to you :)